spec/index.html

```diff
@@ -733,125 +733,139 @@ <h3>Literals</h3>
-      the <a>datatype IRI</a> because it always equals either
-      <code>http://www.w3.org/1999/02/22-rdf-syntax-ns#langString</code> (<code>rdf:langString</code>)
-      or <code>http://www.w3.org/1999/02/22-rdf-syntax-ns#dirLangString</code> (<code>rdf:dirLangString</code>), respectively.</p>
-
-    <p>The <dfn>literal value</dfn> associated with a <a>literal</a> is:</p>
-
-    <ul>
-      <li>If the literal is a <a>language-tagged string</a>,
-        then the literal value is a pair consisting of its <a>lexical form</a>
-        and its <a>language tag</a>, in that order.</li>
-      <li>If the literal is a <a>directional language-tagged string</a>, then the literal value is
-        a tuple of its <a>lexical form</a>, its <a>language tag</a>, and its <a>base direction</a>,
-        likewise in that order.</li>
-      <li>If the literal's <a>datatype</a> is handled by an RDF implmentation,
-        <ul>
-          <li>if the literal's <a>lexical form</a> is in the <a>lexical space</a>
-            of the <a>datatype</a>, then the literal value is the result of applying
-            the <a>lexical-to-value mapping</a> of the datatype to the
-            <a>lexical form</a>.</li>
-          <li>otherwise, the literal is <dfn data-lt-no-plural>ill-typed</dfn> and no literal value can be
-             associated with the literal. Such a case produces a semantic
-             inconsistency but is not <em>syntactically</em> ill-formed.
-             Implementations SHOULD accept [=ill-typed=] literals and produce RDF
-             graphs from them. Implementations MAY produce warnings when
-             encountering [=ill-typed=] literals.</li>
-        </ul>
-      </li>
-      <li>If the literal's <a>datatype IRI</a> is <em>not</em>
-        handled by an RDF implementation, then the literal value is
-        not defined by this specification. Implementations SHOULD accept
-        literals with unknown datatype IRIs and produce RDF graphs from them.
-      </li>
-    </ul>
+    </p>
 
     <p><dfn data-local-lt="term-equal">Literal term equality</dfn>:
-      Two literals are term-equal (the same <a>RDF literal</a>)
+      two literals are term-equal (the same <a>RDF term</a>)
       if and only if:</p>
 
     <ul>
-      <li>the two <a>lexical forms</a> compare equal</li>
-      <li>the two <a>datatype IRIs</a> compare equal</li>
-      <li>the two <a>language tags</a> (if any) compare equal</li>
-      <li>the two <a>base directions</a> (if any) compare equal</li>
+      <li>the two <a>lexical forms</a> compare equal,</li>
+      <li>the two <a>datatype IRIs</a> compare equal,</li>
+      <li>the two <a>language tags</a> are either both absent, or both present and compare equal,</li>
+      <li>the two <a>base directions</a> are either both absent, both `ltr`, or both `rtl`.</li>
     </ul>
-    <p>Comparison is performed using 
+    <p>Comparison of the [=lexical forms=] and of the [=datatype IRIs=] is performed using
```

Comment:
For the datatype IRIs, shouldn't this better be covered by [IRI equality](https://w3c.github.io/rdf-concepts/spec/#dfn-iri-equality)?

Reply:
fair point. I reused existing language, which didn't mention IRI equality.  This is equivalent, because IRI equality is also based on string comparison, but this would be clearer.

Reply:
@hartig are you happy to merge this PR and potentially raise a separate issue for the point above? Otherwise, could you suggest a change? thx

Reply:
@pchampin please merge and I will create a separate PR for this issue afterwards.